semgrep.opam

```diff
@@ -55,7 +55,7 @@ depends: [
   "parmap"
   "semver"
   "timedesc" {>= "2.0.0"}
-  "lsp" {= "1.15.1-5.0"}
+  "lsp" {= "1.20.0"}
```

Comment:
This is actually auto-generated. Update this here: https://github.com/semgrep/semgrep/blob/d08f4320e7acf6ff7b152e937458e8044ce26e91/dune-project#L507

and then run `make semgrep.opam`.